Documentation/proposals/202212-scrape-config.md

```diff
@@ -0,0 +1,106 @@
+# ScrapeConfig CRD
+
+* Owners:
+  * [xiu](https://github.com/xiu)
+* Related Tickets:
+  * [#2787](https://github.com/prometheus-operator/prometheus-operator/issues/2787)
+  * [#3447](https://github.com/prometheus-operator/prometheus-operator/issues/3447)
+* Other docs:
+  * n/a
+
+This document aims at creating a lower level `ScrapeConfig` Custom Resource Definition that defines additional scrape configurations the Kubernetes way.
+
+# Why
```

Comment:
I think that it'd be helpful if you provide more details about your specific use case @xiu 
#2787 was initially about Kubernetes node service discovery which could be mentioned here too. I'll go through the existing issues to see if other use cases are worth mentioning.

Reply:
I've added three use cases to the document.